index.html

```diff
@@ -0,0 +1,1117 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <main class="main">
+        <nav class="nav-bar nav-bar-h">
+            <div class="tnw-logo tnw-logo-h">
+                <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="tnw-logo">
+            </div>
+            <div class="right-nav">
+                <div class="top-elements">
+                    <div class="top-elements-right">
+                        <a href="">News</a>
+                        <a href="">Events</a>
+                        <a href="">Business</a>
+                        <a href="">AMAs</a>
+                        <a href="">Spaces</a>
+                        <a href="">Terms & Conditions</a>
+                    </div>
+                    <div class="top-elements-left">
+                        <a href="">About</a>
+                        <a href="">Advertise</a>
+                        <a href="">Jobs</a>
+                        <a href="">Contact</a>
+                    </div>
+                </div>
+                <div class="right-elements">
+                    <div class="mid-div-left">
+                        <a href="">LATEST</a>
+                        <a href="">HARD FORK</a>
+                        <a href="">PLUGGED</a>
+                        <a href="">README</a>
+                        <a href="">GROWTH &amp QUARTERS</a>
+                    </div>
```

Comment:
[Required] We have to fix the HTML validator error.
![Screenshot from 2020-06-02 16-59-50](https://user-images.githubusercontent.com/37747619/83514400-4df2fe00-a4f3-11ea-87d5-dc582f52dd0f.png)